migrations/20220406123607_USER_TABLE.go

```diff
@@ -0,0 +1,45 @@
+// nolint:dupl //Cannot use same test for different migrations
```

Comment:
Same here. Remove  automated comments.

Reply:
not an automated comment. Had to add it because linter was giving an error saying these lines are duplicate. Asked my colleagues, seems like it is safe to ignore this error here. If you are aware of any other way of resolving it, let me know.

Reply:
Okay. Let's leave it for now. We can fix it later once we're aware of the right practise.